<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1090</width>
    <height>702</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QTabWidget" name="tabWidget">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>0</y>
      <width>931</width>
      <height>641</height>
     </rect>
    </property>
    <property name="mouseTracking">
     <bool>false</bool>
    </property>
    <property name="focusPolicy">
     <enum>Qt::NoFocus</enum>
    </property>
    <property name="acceptDrops">
     <bool>false</bool>
    </property>
    <property name="tabPosition">
     <enum>QTabWidget::North</enum>
    </property>
    <property name="tabShape">
     <enum>QTabWidget::Rounded</enum>
    </property>
    <property name="currentIndex">
     <number>0</number>
    </property>
    <property name="iconSize">
     <size>
      <width>20</width>
      <height>20</height>
     </size>
    </property>
    <property name="elideMode">
     <enum>Qt::ElideMiddle</enum>
    </property>
    <property name="tabBarAutoHide">
     <bool>false</bool>
    </property>
    <widget class="QWidget" name="create_tab">
     <property name="focusPolicy">
      <enum>Qt::TabFocus</enum>
     </property>
     <attribute name="title">
      <string>Create Wallet</string>
     </attribute>
     <widget class="QTabWidget" name="tabWidget_derivation">
      <property name="geometry">
       <rect>
        <x>20</x>
        <y>80</y>
        <width>881</width>
        <height>421</height>
       </rect>
      </property>
      <property name="tabPosition">
       <enum>QTabWidget::North</enum>
      </property>
      <property name="tabShape">
       <enum>QTabWidget::Triangular</enum>
      </property>
      <property name="currentIndex">
       <number>0</number>
      </property>
      <widget class="QWidget" name="tab_10">
       <attribute name="title">
        <string>Generate Wallet</string>
       </attribute>
       <widget class="QLabel" name="purpose_label_2">
        <property name="geometry">
         <rect>
          <x>240</x>
          <y>140</y>
          <width>61</width>
          <height>16</height>
         </rect>
        </property>
        <property name="text">
         <string>Language</string>
        </property>
       </widget>
       <widget class="QComboBox" name="coin_comboBox">
        <property name="enabled">
         <bool>true</bool>
        </property>
        <property name="geometry">
         <rect>
          <x>340</x>
          <y>160</y>
          <width>81</width>
          <height>22</height>
         </rect>
        </property>
        <property name="mouseTracking">
         <bool>true</bool>
        </property>
        <property name="focusPolicy">
         <enum>Qt::StrongFocus</enum>
        </property>
        <property name="acceptDrops">
         <bool>true</bool>
        </property>
        <property name="styleSheet">
         <string notr="true"/>
        </property>
        <property name="currentText">
         <string/>
        </property>
        <property name="currentIndex">
         <number>-1</number>
        </property>
        <property name="insertPolicy">
         <enum>QComboBox::NoInsert</enum>
        </property>
        <property name="placeholderText">
         <string notr="true">Currency</string>
        </property>
        <property name="frame">
         <bool>false</bool>
        </property>
        <item>
         <property name="text">
          <string>Bitcoin</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>Litecoin</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>Dogecoin</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>Ethereum</string>
         </property>
        </item>
       </widget>
       <widget class="QPushButton" name="generate_button">
        <property name="geometry">
         <rect>
          <x>240</x>
          <y>270</y>
          <width>75</width>
          <height>24</height>
         </rect>
        </property>
        <property name="text">
         <string>PushButton</string>
        </property>
       </widget>
       <widget class="QLineEdit" name="passphrase_field">
        <property name="geometry">
         <rect>
          <x>240</x>
          <y>200</y>
          <width>231</width>
          <height>22</height>
         </rect>
        </property>
        <property name="text">
         <string>Passphrase</string>
        </property>
       </widget>
       <widget class="QComboBox" name="language_comboBox">
        <property name="geometry">
         <rect>
          <x>240</x>
          <y>160</y>
          <width>81</width>
          <height>22</height>
         </rect>
        </property>
        <property name="placeholderText">
         <string>Language</string>
        </property>
        <item>
         <property name="text">
          <string>Chinese simplified</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>Chinese traditional</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>English</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>French</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>Italian</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>Japanese</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>Korean</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>Spanish</string>
         </property>
        </item>
       </widget>
       <widget class="QLineEdit" name="passphrase_field_2">
        <property name="geometry">
         <rect>
          <x>240</x>
          <y>240</y>
          <width>231</width>
          <height>22</height>
         </rect>
        </property>
        <property name="text">
         <string>Give a name to your wallet</string>
        </property>
       </widget>
       <widget class="QLabel" name="purpose_label_4">
        <property name="geometry">
         <rect>
          <x>340</x>
          <y>140</y>
          <width>61</width>
          <height>16</height>
         </rect>
        </property>
        <property name="text">
         <string>Currency</string>
        </property>
       </widget>
      </widget>
      <widget class="QWidget" name="tab_9">
       <attribute name="title">
        <string>Get new address</string>
       </attribute>
       <widget class="QComboBox" name="comboBox">
        <property name="geometry">
         <rect>
          <x>320</x>
          <y>120</y>
          <width>91</width>
          <height>22</height>
         </rect>
        </property>
        <property name="placeholderText">
         <string>Account ID</string>
        </property>
       </widget>
       <widget class="QComboBox" name="purpose_comboBox">
        <property name="geometry">
         <rect>
          <x>320</x>
          <y>150</y>
          <width>91</width>
          <height>22</height>
         </rect>
        </property>
        <property name="currentIndex">
         <number>-1</number>
        </property>
        <property name="placeholderText">
         <string>Change</string>
        </property>
        <item>
         <property name="text">
          <string extracomment="Can only be used for recieving funds">Internal</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string extracomment="Can only be used for sending funds">External</string>
         </property>
        </item>
       </widget>
       <widget class="QLabel" name="purpose_label_5">
        <property name="geometry">
         <rect>
          <x>320</x>
          <y>70</y>
          <width>61</width>
          <height>16</height>
         </rect>
        </property>
        <property name="text">
         <string>Change</string>
        </property>
       </widget>
       <widget class="QLabel" name="purpose_label_6">
        <property name="geometry">
         <rect>
          <x>210</x>
          <y>200</y>
          <width>331</width>
          <height>20</height>
         </rect>
        </property>
        <property name="text">
         <string>External - the address can only be used for receiving funds</string>
        </property>
       </widget>
       <widget class="QLabel" name="purpose_label">
        <property name="geometry">
         <rect>
          <x>210</x>
          <y>180</y>
          <width>331</width>
          <height>20</height>
         </rect>
        </property>
        <property name="text">
         <string>Internal - the address can only be used for receiving funds</string>
        </property>
       </widget>
       <widget class="QLabel" name="label">
        <property name="geometry">
         <rect>
          <x>230</x>
          <y>40</y>
          <width>311</width>
          <height>16</height>
         </rect>
        </property>
        <property name="text">
         <string>There can only exist 10 addresses per account and change</string>
        </property>
       </widget>
       <widget class="QPushButton" name="pushButton_13">
        <property name="geometry">
         <rect>
          <x>320</x>
          <y>230</y>
          <width>91</width>
          <height>24</height>
         </rect>
        </property>
        <property name="text">
         <string>Generate</string>
        </property>
       </widget>
       <widget class="QComboBox" name="comboBox_2">
        <property name="geometry">
         <rect>
          <x>320</x>
          <y>90</y>
          <width>91</width>
          <height>22</height>
         </rect>
        </property>
        <property name="placeholderText">
         <string>Wallet ID</string>
        </property>
       </widget>
       <widget class="QWidget" name="verticalLayoutWidget">
        <property name="geometry">
         <rect>
          <x>0</x>
          <y>90</y>
          <width>321</width>
          <height>80</height>
         </rect>
        </property>
        <layout class="QVBoxLayout" name="verticalLayout"/>
       </widget>
      </widget>
     </widget>
     <widget class="QLabel" name="counter_wallets">
      <property name="geometry">
       <rect>
        <x>90</x>
        <y>30</y>
        <width>191</width>
        <height>31</height>
       </rect>
      </property>
      <property name="text">
       <string>Active wallets</string>
      </property>
     </widget>
     <widget class="QLabel" name="counter_accounts">
      <property name="geometry">
       <rect>
        <x>370</x>
        <y>30</y>
        <width>181</width>
        <height>41</height>
       </rect>
      </property>
      <property name="text">
       <string>Active accounts</string>
      </property>
     </widget>
    </widget>
    <widget class="QWidget" name="view_tab">
     <attribute name="title">
      <string>View Wallets</string>
     </attribute>
     <widget class="QTableWidget" name="vw_tableWidget">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>60</y>
        <width>841</width>
        <height>192</height>
       </rect>
      </property>
      <column>
       <property name="text">
        <string>Currency</string>
       </property>
       <property name="font">
        <font>
         <pointsize>12</pointsize>
         <bold>true</bold>
        </font>
       </property>
      </column>
      <column>
       <property name="text">
        <string>Path</string>
       </property>
       <property name="font">
        <font>
         <pointsize>12</pointsize>
         <bold>true</bold>
        </font>
       </property>
      </column>
      <column>
       <property name="text">
        <string>Address</string>
       </property>
       <property name="font">
        <font>
         <pointsize>12</pointsize>
         <bold>true</bold>
        </font>
       </property>
      </column>
      <column>
       <property name="text">
        <string>Private Key</string>
       </property>
       <property name="font">
        <font>
         <pointsize>12</pointsize>
         <bold>true</bold>
        </font>
       </property>
      </column>
      <column>
       <property name="text">
        <string>Change</string>
       </property>
       <property name="font">
        <font>
         <pointsize>12</pointsize>
         <bold>true</bold>
        </font>
       </property>
      </column>
      <column>
       <property name="text">
        <string>Account no</string>
       </property>
       <property name="font">
        <font>
         <pointsize>12</pointsize>
         <bold>true</bold>
        </font>
       </property>
      </column>
      <column>
       <property name="text">
        <string>Fingerprint</string>
       </property>
      </column>
     </widget>
     <widget class="QTabWidget" name="tabWidget_3">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>270</y>
        <width>831</width>
        <height>331</height>
       </rect>
      </property>
      <property name="currentIndex">
       <number>0</number>
      </property>
      <widget class="QWidget" name="tab_5">
       <attribute name="title">
        <string>Tab 1</string>
       </attribute>
       <widget class="QPushButton" name="pushButton_5">
        <property name="geometry">
         <rect>
          <x>10</x>
          <y>150</y>
          <width>75</width>
          <height>24</height>
         </rect>
        </property>
        <property name="text">
         <string>Send</string>
        </property>
       </widget>
       <widget class="QPushButton" name="pushButton_6">
        <property name="geometry">
         <rect>
          <x>110</x>
          <y>150</y>
          <width>81</width>
          <height>24</height>
         </rect>
        </property>
        <property name="text">
         <string>Check Fee</string>
        </property>
       </widget>
       <widget class="QLineEdit" name="lineEdit_18">
        <property name="geometry">
         <rect>
          <x>20</x>
          <y>60</y>
          <width>191</width>
          <height>22</height>
         </rect>
        </property>
        <property name="text">
         <string>Receiver address</string>
        </property>
       </widget>
       <widget class="QLineEdit" name="lineEdit_19">
        <property name="geometry">
         <rect>
          <x>240</x>
          <y>60</y>
          <width>113</width>
          <height>22</height>
         </rect>
        </property>
        <property name="text">
         <string>Balance</string>
        </property>
       </widget>
       <widget class="QComboBox" name="comboBox_5">
        <property name="geometry">
         <rect>
          <x>380</x>
          <y>60</y>
          <width>91</width>
          <height>22</height>
         </rect>
        </property>
        <property name="placeholderText">
         <string>Currency</string>
        </property>
       </widget>
       <widget class="QComboBox" name="comboBox_6">
        <property name="geometry">
         <rect>
          <x>490</x>
          <y>60</y>
          <width>121</width>
          <height>22</height>
         </rect>
        </property>
        <property name="placeholderText">
         <string>From Address</string>
        </property>
       </widget>
      </widget>
      <widget class="QWidget" name="tab_6">
       <attribute name="title">
        <string>Tab 2</string>
       </attribute>
      </widget>
     </widget>
    </widget>
    <widget class="QWidget" name="add_tab">
     <attribute name="title">
      <string>Add External Wallet</string>
     </attribute>
     <widget class="QTabWidget" name="tabWidget_2">
      <property name="geometry">
       <rect>
        <x>30</x>
        <y>20</y>
        <width>851</width>
        <height>561</height>
       </rect>
      </property>
      <property name="tabPosition">
       <enum>QTabWidget::South</enum>
      </property>
      <property name="currentIndex">
       <number>1</number>
      </property>
      <widget class="QWidget" name="tab_2">
       <attribute name="title">
        <string>Classic Wallet</string>
       </attribute>
       <widget class="QTableWidget" name="ec_tableWidget">
        <property name="geometry">
         <rect>
          <x>90</x>
          <y>170</y>
          <width>601</width>
          <height>192</height>
         </rect>
        </property>
        <column>
         <property name="text">
          <string>Currency</string>
         </property>
         <property name="font">
          <font>
           <pointsize>12</pointsize>
           <bold>true</bold>
          </font>
         </property>
        </column>
        <column>
         <property name="text">
          <string>Public Key</string>
         </property>
         <property name="font">
          <font>
           <pointsize>12</pointsize>
           <bold>true</bold>
          </font>
         </property>
        </column>
        <column>
         <property name="text">
          <string>Private Key</string>
         </property>
         <property name="font">
          <font>
           <pointsize>12</pointsize>
           <bold>true</bold>
          </font>
         </property>
        </column>
       </widget>
       <widget class="QLineEdit" name="ec_pubk_field">
        <property name="geometry">
         <rect>
          <x>410</x>
          <y>30</y>
          <width>151</width>
          <height>22</height>
         </rect>
        </property>
        <property name="text">
         <string>Public Key</string>
        </property>
       </widget>
       <widget class="QLineEdit" name="ec_coin_field">
        <property name="geometry">
         <rect>
          <x>220</x>
          <y>30</y>
          <width>151</width>
          <height>22</height>
         </rect>
        </property>
        <property name="text">
         <string>Currency</string>
        </property>
       </widget>
       <widget class="QPushButton" name="ec_insert_btn">
        <property name="geometry">
         <rect>
          <x>350</x>
          <y>130</y>
          <width>75</width>
          <height>24</height>
         </rect>
        </property>
        <property name="text">
         <string>Insert</string>
        </property>
       </widget>
       <widget class="QLineEdit" name="ec_privk_field">
        <property name="geometry">
         <rect>
          <x>410</x>
          <y>90</y>
          <width>151</width>
          <height>22</height>
         </rect>
        </property>
        <property name="text">
         <string>Private Key</string>
        </property>
       </widget>
      </widget>
      <widget class="QWidget" name="tab">
       <attribute name="title">
        <string>HD Wallet</string>
       </attribute>
       <widget class="QTableWidget" name="ehd_tableWidget">
        <property name="geometry">
         <rect>
          <x>0</x>
          <y>150</y>
          <width>841</width>
          <height>192</height>
         </rect>
        </property>
        <column>
         <property name="text">
          <string>Currency</string>
         </property>
         <property name="font">
          <font>
           <pointsize>12</pointsize>
           <bold>true</bold>
          </font>
         </property>
        </column>
        <column>
         <property name="text">
          <string>Mnemonic</string>
         </property>
         <property name="font">
          <font>
           <pointsize>12</pointsize>
           <bold>true</bold>
          </font>
         </property>
        </column>
        <column>
         <property name="text">
          <string>Language</string>
         </property>
        </column>
        <column>
         <property name="text">
          <string>Passphrase</string>
         </property>
        </column>
       </widget>
       <widget class="QLineEdit" name="ehd_mnemonic_field">
        <property name="geometry">
         <rect>
          <x>270</x>
          <y>30</y>
          <width>151</width>
          <height>22</height>
         </rect>
        </property>
        <property name="text">
         <string>Mnemonic</string>
        </property>
       </widget>
       <widget class="QPushButton" name="ehd_insert_btn">
        <property name="geometry">
         <rect>
          <x>370</x>
          <y>80</y>
          <width>75</width>
          <height>24</height>
         </rect>
        </property>
        <property name="text">
         <string>Insert</string>
        </property>
       </widget>
       <widget class="QLineEdit" name="ehd_passphrase_field">
        <property name="geometry">
         <rect>
          <x>450</x>
          <y>30</y>
          <width>151</width>
          <height>22</height>
         </rect>
        </property>
        <property name="text">
         <string>Passphrase</string>
        </property>
       </widget>
       <widget class="QComboBox" name="ehd_language_comboBox">
        <property name="geometry">
         <rect>
          <x>130</x>
          <y>30</y>
          <width>121</width>
          <height>22</height>
         </rect>
        </property>
        <property name="placeholderText">
         <string>Language</string>
        </property>
        <item>
         <property name="text">
          <string>Chinese simplified</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>Chinese traditional</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>English</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>French</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>Italian</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>Japanese</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>Korean</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>Spanish</string>
         </property>
        </item>
       </widget>
       <widget class="QComboBox" name="ehd_currency_comboBox">
        <property name="enabled">
         <bool>true</bool>
        </property>
        <property name="geometry">
         <rect>
          <x>30</x>
          <y>30</y>
          <width>81</width>
          <height>22</height>
         </rect>
        </property>
        <property name="mouseTracking">
         <bool>true</bool>
        </property>
        <property name="focusPolicy">
         <enum>Qt::StrongFocus</enum>
        </property>
        <property name="acceptDrops">
         <bool>true</bool>
        </property>
        <property name="styleSheet">
         <string notr="true"/>
        </property>
        <property name="currentText">
         <string/>
        </property>
        <property name="currentIndex">
         <number>-1</number>
        </property>
        <property name="insertPolicy">
         <enum>QComboBox::NoInsert</enum>
        </property>
        <property name="placeholderText">
         <string notr="true">Currency</string>
        </property>
        <property name="frame">
         <bool>false</bool>
        </property>
        <item>
         <property name="text">
          <string>Bitcoin</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>Litecoin</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>Dogecoin</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>Ethereum</string>
         </property>
        </item>
       </widget>
      </widget>
     </widget>
    </widget>
   </widget>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
